docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
+| **Category** | **Definition** | **KPI** |
+|----------|-----------|-----|
+| Data Completeness | Measures the extent to which all required data fields are present in the dataset and tracks the overall data completeness trend over a specified period.| Percentage of data fields that are complete and the overall data completeness over time. |
+| Data Quality Performance | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
+|Data Ingestion Frequency | Measures how often data is ingested into the system. | Number of data ingestion events per unit of time (daily, weekly, monthly, quarterly, annually). |
```

Comment:
This raises a question of recommended target. Having a number is only useful if you know whether that's good or bad and can make a plan to get it green. Most targets will merit an "it depends" answer, so maybe we can at least speak to how they can decide their target. In this case, we can say something about basing frequency on how often the data is being used (e.g., daily to anomaly detection or monthly for monthly business reviews).